helix-term/src/keymap.rs

```diff
@@ -361,7 +362,14 @@ impl Keymaps {
     pub fn get(&mut self, mode: Mode, key: KeyEvent) -> KeymapResult {
         // TODO: remove the sticky part and look up manually
         let keymaps = &*self.map();
-        let keymap = &keymaps[&mode];
+
+        // Ideally this would be:
+        // let keymap = keymaps.get(&mode).ok_or(KeymapResult::NotFound)?;
```

Comment:
I think this comment can be removed - I'm not sure it makes sense to refactor `KeymapResult` into a `Result`

Reply:
Noted, I removed the comment